geopandas/accessors.py

```diff
@@ -0,0 +1,28 @@
+"""
+Accessors for accessing GeoPandas functionality via pandas extension dtypes
+"""
+
+import pandas.api.extensions
+
+import geopandas.geoseries
+from geopandas.array import GeometryDtype
+
+
+@pandas.api.extensions.register_series_accessor("geo")
+class GeoSeriesAccessor:
+    """Series.geo accessor to expose GeoSeries methods on pandas Series.
+
+    Parameters
+    ----------
+    series : pandas.Series
+        A Series with geometry dtype.
+    """
+
+    def __init__(self, series):
+        if not isinstance(series.dtype, GeometryDtype):
+            raise AttributeError("Can only use .geo accessor with geometry values")
```

Comment:
can you add a test covering this? (it seems that's not showing in CI right now, we've had a few issues with it)

Reply:
Added in https://github.com/geopandas/geopandas/pull/3272/commits/8aa87144eff87bacd415346778bb60be729b1ba7

Full coverage of `geopandas/accessors.py` now.

```
---------- coverage: platform darwin, python 3.11.2-final-0 ----------
Name                                Stmts   Miss  Cover
-------------------------------------------------------
geopandas/__init__.py                  19      0   100%
geopandas/_compat.py                   36      4    89%
geopandas/_config.py                   42      2    95%
geopandas/_decorator.py                16      0   100%
geopandas/accessors.py                 11      0   100%
geopandas/array.py                    708     26    96%
geopandas/base.py                     362     15    96%
geopandas/conftest.py                  25      3    88%
geopandas/datasets/__init__.py         11      0   100%
geopandas/explore.py                  281    268     5%
geopandas/geodataframe.py             481     30    94%
geopandas/geoseries.py                246     25    90%
geopandas/io/__init__.py                0      0   100%
geopandas/io/_geoarrow.py             264    264     0%
geopandas/io/_pyarrow_hotfix.py        27     27     0%
geopandas/io/arrow.py                 273    232    15%
geopandas/io/file.py                  335    229    32%
geopandas/io/sql.py                   134    118    12%
geopandas/io/util.py                   45     37    18%
geopandas/plotting.py                 326    305     6%
geopandas/sindex.py                    92      8    91%
geopandas/testing.py                   97      9    91%
geopandas/tools/__init__.py             6      0   100%
geopandas/tools/_random.py             26      6    77%
geopandas/tools/_show_versions.py      83     22    73%
geopandas/tools/clip.py                75     43    43%
geopandas/tools/geocoding.py           58     48    17%
geopandas/tools/hilbert_curve.py       74      2    97%
geopandas/tools/overlay.py            164      8    95%
geopandas/tools/sjoin.py              180     21    88%
geopandas/tools/util.py                17     12    29%
-------------------------------------------------------
TOTAL                                4514   1764    61%
```